### PR TITLE
Added quick utility script for making a new rule

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
@Res42's efforts in #1085 reminded me I've been meaning to write this for a little while... should be helpful in setting up new rules, in lieu of #813. Thoughts?

### CODE REVIEW

File: script/new-converter.js
Comment (JoshuaKGoldberg) [inline script/new-converter.js:63]: This script doesn't (yet?) add to `ruleConverters.ts`. I ran out of energy :smile: but it would be a good followup... or I might tackle it soon.

**Comment by Res42** [comment]
I generated my rules too, so here are some thoughts.

File: script/new-converter.js
Comment (Res42) [inline script/new-converter.js:21]: Currently there is a plugin that has a prefix and it is not included in the plugin list in converters: `@typescript-eslint`. It is assumed that plugin is always installed I think?

Comment (JoshuaKGoldberg) [reply script/new-converter.js:21]: Ha, true. I was just thinking of the react-a11y ones!

File: script/new-converter.js
Comment (JoshuaKGoldberg) [reply script/new-converter.js:93]: I will after merging in from #1102

**Review by KingDarBoja — APPROVED** [review]
LGTM, and I trust it works even if there are no tests 😆

**Pull request merged**
